[2024-11-02T13:31:24.964+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: schema_dag.submit_schemas scheduled__2024-06-04T00:00:00+00:00 [queued]>
[2024-11-02T13:31:24.980+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: schema_dag.submit_schemas scheduled__2024-06-04T00:00:00+00:00 [queued]>
[2024-11-02T13:31:24.981+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-11-02T13:31:25.012+0000] {taskinstance.py:1380} INFO - Executing <Task(PinotSchemaSubmitOperator): submit_schemas> on 2024-06-04 00:00:00+00:00
[2024-11-02T13:31:25.020+0000] {standard_task_runner.py:57} INFO - Started process 3657 to run task
[2024-11-02T13:31:25.026+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'schema_dag', 'submit_schemas', 'scheduled__2024-06-04T00:00:00+00:00', '--job-id', '1343', '--raw', '--subdir', 'DAGS_FOLDER/schema_dag.py', '--cfg-path', '/tmp/tmpapxl6xb5']
[2024-11-02T13:31:25.046+0000] {standard_task_runner.py:85} INFO - Job 1343: Subtask submit_schemas
[2024-11-02T13:31:25.184+0000] {task_command.py:415} INFO - Running <TaskInstance: schema_dag.submit_schemas scheduled__2024-06-04T00:00:00+00:00 [running]> on host 5e812a6a7c02
[2024-11-02T13:31:25.312+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='minhtranquang' AIRFLOW_CTX_DAG_ID='schema_dag' AIRFLOW_CTX_TASK_ID='submit_schemas' AIRFLOW_CTX_EXECUTION_DATE='2024-06-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-04T00:00:00+00:00'
[2024-11-02T13:31:25.314+0000] {pinot_schema_operator.py:33} ERROR - An error occured: 'function' object has no attribute 'glob'
[2024-11-02T13:31:25.345+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=schema_dag, task_id=submit_schemas, execution_date=20240604T000000, start_date=20241102T133124, end_date=20241102T133125
[2024-11-02T13:31:25.402+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-11-02T13:31:25.528+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
